ibm-messaging-mq-cloud-showcase-app/frontend/Dockerfile

```diff
@@ -46,7 +46,8 @@ RUN yarn run build
 FROM nginx:stable-alpine3.17 as frontendImage
 
 #Set nginx configuration as per application requirements
-COPY nginx.conf.template /etc/nginx/conf.d/default.conf.template
+COPY nginx.conf.template /etc/nginx/conf.d
+COPY nginx.conf.proxy.template /etc/nginx/conf.d
```

Comment:
Why are there 2 copies to `/etc/nginx/conf.d` ?

Should this be a conditional copy depending on whether FE is being used as a proxy?

Reply:
Since the Env variables are not available during the build phase , I am copying both the templates to the container and serving the template based on REACT_APP_FE_AS_PROXY in docker-entrypoint.sh.